File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.java
class Solution {
    public int longestConsecutive(int[] nums) {
        int cnt=1,lon=1;
        if(nums.length==0) return 0;
        Arrays.sort(nums);
        for(int i=1;i<nums.length;i++){
            if(nums[i]!=nums[i-1]){
                if(nums[i]==nums[i-1]+1) cnt++;
                else{
                lon=Math.max(lon,cnt);
                cnt=1;
            }
            }
            
            
        }
        return Math.max(lon,cnt);
    }
}

File: Data Structures & Algorithms/duplicate-integer/submission-2.py
class Solution:
    def hasDuplicate(self, nums: List[int]) -> bool:
        mp={}
        for i in range(0,len(nums),1):
            mp[nums[i]]=mp.get(nums[i],0)+1

        for val in mp.values():
            if val>1:
                return True

        return False


            
        